CHANGELOG.md

```diff
@@ -13,39 +13,94 @@ All notable changes to this project will be documented in this file.
+  collaborate around all things related to PyMedPhys.
 
-### Breaking public API changes
+### Breaking changes
 
-(Won't truly be stable until after a 1.0.0 release)
+* **Developer facing only**: The `--live` parameter within `pymedphys dev docs`
+  has been removed.
 
 ### New features and enhancements
 
-* `pymedphys dicom listen` stores incoming DICOM objects in a directory hierarchy: PatientID/Study Instance UID/Series Instance UID
-aligning with the DICOM Q/R hierarchy, and more suitable for use with tools like dicompyler and OnkoDICOM. NOTE: This changes where a 3rd party or in-house program would expect to find the DICOM data on the file system compared to the previous release.
-
-* Linked to Discourse group witin tutorials
-* Discourse commenting now available directly within the hosted documentation
+* `pymedphys dicom listen` stores incoming DICOM objects in a directory
+  hierarchy: PatientID/Study Instance UID/Series Instance UID aligning with the
+  DICOM Q/R hierarchy, and more suitable for use with tools like dicompyler and
+  OnkoDICOM. See PR [#1208](https://github.com/pymedphys/pymedphys/pull/1208)
+  for more details.
 
-* improved layout and fonts
-* improved demonstrations and tutorials using Jupyter Book
-*  app.pymedphys.com runs live against the main branch of pymedphys
+```{note}
+This `pymedphys dicom listen` adjustment changes where a 3rd party or in-house
+program would expect to find the DICOM data on the file system compared to the
+previous release.
 
-* increased docstring coverage of public functions
+Please raise an issue on GitHub
+<https://github.com/pymedphys/pymedphys/issues/new> if you believe changes like
+this in the future should be considered a breaking change.
+```
 
-* Installation on MacOS (Intel) has been simplified and is now the same as for other platforms, thanks to work by the new pymssql maintainers.
+* Documentation now uses the new [Jupyter Book](https://jupyterbook.org/) tool
+  from the [Executable Book Project](https://executablebooks.org/). Among other
+  things this has enabled:
+  * Live running of notebook documentation using
+    [Thebe](https://thebelab.readthedocs.io/)
+  * Discourse commenting now available directly within the hosted documentation
+  * The ability to utilise the expanded
+    [MyST](https://jupyterbook.org/content/myst.html) Documentation formatting.
+* Increased docstring coverage of public functions
+* Installation on MacOS (Intel) has been simplified and is now the same as for
+  other platforms, thanks to [@termim](https://github.com/termim) who has taken
+  on the mantle of maintaining `pymssql`. See PR
+  [pymssql#677](https://github.com/pymssql/pymssql/pull/677) for more details.
+* Streamlit development now supports reloading during development across the
+  whole PyMedPhys library. See PR
+  [#1202](https://github.com/pymedphys/pymedphys/pull/1202) for more details.
+* A pre-release binary has been built and CI infrastructure around it has begun
```

Comment:
I think if it isn't in the release, it's best not to mention it.  
Otherwise it's a "future" (which is understandable when one has to market one's product, but I don't think that's the case here).
If it's in a pre-release, that should be in the notes for the pre-release (dev/beta/alpha, whatever), and then show up again in the actual release.

Reply:
My thoughts here was that since the binary cause a range of flow on changes (eg `python -m pymedphys`) it was worth at least mentioning in a low-profile fashion. It is helpful at least as a record when "looking back" a year from now. Let's say I wanted to find out what tag has the first instance of this in it, I have found that kind of flag as displayed here quite interesting.

Reply:
I'm OK with it.  You have a strong preference in one direction, I have a mild preference in the other.

Reply:
> I'm OK with it. You have a strong preference in one direction, I have a mild preference in the other.

Thanks Stuart 🙂 as our community grows as a group we're only going to have more diverging preferences around the place. It's going to be a learning curve for me on how to appropriately navigate that.